apm-configuration/python.mdx

```diff
@@ -61,61 +66,66 @@ Refer more about Docker bridge networking [here](https://docs.docker.com/network
 kubectl get service --all-namespaces | grep mw-service
 ```
 
-Then add the following environment variable to your application deployment YAML file. 
+Then add the following environment variable to your application deployment YAML file:
 ```bash Shell
 MW_AGENT_SERVICE=mw-service.mw-agent-ns.svc.cluster.local
 ```
 
 ## Step 4: Capture Application Data
 
-### Traces
-Distributed tracing will be automatically enabled upon completing Step 4: Run Application. 
+### Enable Custom Logs
 
-### Metrics
-Enable metrics by writing the following line just after the tracker initialization from Step 2.
-```python Python 3
-tracker.collect_metrics()
-```
+To ingest custom logs, utilize the following functions based on desired log severity levels:
 
-### Application & Custom Logs
-Enable application logs by writing the following line after the tracker initialization from Step 2. 
-```python Python 3
-tracker.collect_logs()
-```
-To ingest custom logs, utilize the following functions based on desired log severity levels.
 ```python Python 3
 logging.info(“<info sample>”)
-logging.warn(“<warn sample>”)
-logging.error(“<error sample>”)
+logging.warning("Sample Warning Log.") 
+logging.error("Sample Error Log.", extra={'tester': 'Alex'})
 ```
 
-### Profiling
-Enable profiling by writing the following line after the tracker initialization from Step 2.
+### Stack Errors
+Use `tracker.record_error()` method to record a stack trace when an error occurs:
+
 ```python Python 3
-tracker.collect_profiling()
+try:
+    not_possible = 12/0
+except ZeroDivisionError as e:
+    tracker.record_error(e)
 ```
 
-### Stack Traces
-Use the tracker.record_error(e) method to record a stack trace when an error occurs. See an example of this method below. 
-```python Python 3
-randomList = ['a', 0, 2]
-
-for entry in randomList:
-    try:
-         r = 1/int(entry)
-         break
-    except Exception as e:
-         tracker.record_error(e)
+### Traces, Metrics, Logs, & Profiling Data
```

Comment:
Rename to "Setup .ini File" and have opening sentence call out how the service_name and access_token are required for the tracker to send data to MW.